### ISSUE TITLE
Debugging extension in dev environment doesn't unpack current backend build

### ISSUE THREAD

**Issue by arosien** [apache/daffodil-vscode#471]
Debugging the extension launches `yarn watch` which does a `sbt universal:packageBin` which builds the backend, but the newly built backend isn't installed into the local XDG directory, so whatever release was previously installed there (from an official release, etc.) will be used. This causes confusion if you are expecting the existing backend code to be run.

Workaround: delete your `daffodil-dap` XDG directory and the extension will look in the current directory to install the local backend. (FYI, the tests unpack the current backend build into a custom directory and point the extension to there.)

Fix: do this install during the extension build.

**Comment by Shanedell** [comment]
Yeah so all of the jars would still be jars and not be extracted. So what I am referring to is we currently bundle zip file created from running `sbt universal:packageBin`. I think we should actually extract that zip and bundle the folder, which has a name like `daffodil-debugger-${daffodil_version}-${extension_version}`. Inside of that folder it has a `bin` and `lib` folder for running the debugger and all of the jars needed, those would stay untouched.

**Comment by shanedell** [comment]
Possibly add a flag to `.vscode/launch.json` something like `devMode: (true/false)`. This will make it so in the future if we need to be able to do one thing during development and not in production. This would make it so we use one variable instead of a bunch of different one for each item that comes out.
